client/src/App.css

```
@@ -42,6 +42,12 @@
   justify-content: center;
   font-size: calc(10px + 2vmin);
   padding-bottom: 10px;
+  
+}
+.nav-container > a {
```

There should be a blank line in between these.
```suggestion

.nav-container > a {
```